pkg/target/variableref_test.go

```diff
@@ -342,32 +342,6 @@ resources:
 - ../o1
 - ../o2
 `)
-
-	const presumablyDesired = `
```

Comment:
ditto

Reply:
unused var.

Reply:
reverted the file.  will figure out the intent some other time.